src/pages/HttpBin/index.js

```diff
@@ -1,7 +1,146 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import {
+  Container,
+  Grid,
+  Paper,
+  Button,
+  Typography,
+  CircularProgress,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import axios from 'axios';
+
+import FormInput from '../../components/FormInput';
+import SelectFormInput from '../../components/SelectFormInput';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    maxWidth: '1360px',
+    padding: '10px',
+    margin: '0 auto',
+    boxShadow: 'none',
+  },
+  sendButton: {
+    width: '100%',
+    marginTop: '14px',
+    height: '42px',
+  },
+}));
+
+// ----------------------- EXAMPLES
+const exampleJson = `{"name":"Molecule Man","age":29,"secretIdentity":"Dan Jukes","powers":["Radiation resistance","Turning tiny","Radiation blast"]}`;
+// --------------------------END EXAMPLES
```

Comment:
do you really need an example here?

Reply:
Moved to json file in /data/ update in next commit